apps/pwabuilder/src/script/components/publish-pane.ts

```diff
@@ -842,22 +905,22 @@ export class PublishPane extends LitElement {
       let errString = err.stack;
       stack_trace += errString.slice(
         errString.indexOf(" at ") + 1
-      ); 
+      );
       title = errString.split(",")[0]; // first line of error message
       quick_desc = errString.slice(
         errString.indexOf("Details:") + 8,
         errString.indexOf(" at ")
       ); // the quick description they get to read (searchable)
 
     } else if (this.selectedStore === "Android"){
-      title = response.statusText; 
-      stack_trace += response.stack_trace.split("stack:")[1]; 
+      title = response.statusText;
```

Comment:
let's comment this code some just to make it easier to read through.

Reply:
This is actually something we should look into rewriting now that I am thinking about it. Basically, I knew the format of the error messages that return from each stores API and this function breaks down the error block accordingly and displays it in one of our red error banners.